symphony/content/content.publish.php

```diff
@@ -495,7 +495,10 @@ public function __viewIndex()
 
         // Flag filtering
         if (isset($_REQUEST['filter'])) {
-            $filter_stats = new XMLElement('p', '<span>– ' . __('filtered (%d of %d)', array($entries['total-entries'], EntryManager::fetchCount($section_id))) . '</span>', array('class' => 'inactive'));
+            $filter_stats = new XMLElement('p', '<span>– ' . __('%d of %d entries (filtered)', array($entries['total-entries'], EntryManager::fetchCount($section_id))) . '</span>', array('class' => 'inactive'));
+            $this->Breadcrumbs->appendChild($filter_stats);
```

Comment:
You mean line 499? I just thought that öater one might add more conditions, which theoroetically might not append `$filter_stats` (but something different, or nothing). Maybe I over-thought this?.